src/main/java/edu/harvard/iq/dataverse/MapLayerMetadata.java

```diff
@@ -53,6 +53,7 @@
     // ForeignKey to DataFile
     //x@ManyToOne
     // For now, make this unique:  Each DataFile may only have one map
+    //@OneToOne(cascade = {CascadeType.REMOVE, CascadeType.MERGE, CascadeType.PERSIST}) // TODO: Figure out why this doesn't work.
```

Comment:
As discussed at standup, I gave up on this. Calling `DeleteMapLayerMetadataCommand` felt cleaner anyway because there might be other cleanup that needs to happen. @scolapasta and @matthew-a-dunlap plan to discuss this.